packages/react-instantsearch/src/core/createInstantSearchManager.result.test.js

```diff
@@ -91,7 +91,7 @@ describe('createInstantSearchManager', () => {
       });
     });
     describe('on search for facet values', () => {
-      it.only('updates the store and searches', () => {
```

Comment:
Small info: I am waiting for jest to release eslint-plugin-jest to check those mistakes automatically